From 6f751aaa0e949de3abb41596e160f4533ea6ba98 Mon Sep 17 00:00:00 2001
From: unknown <zhouxuelian@zhouxuelian-PC.corp.dhf100.com>
Date: Wed, 29 Apr 2015 19:51:12 +0800
Subject: [PATCH] The file test1.txt has been committed. Update the file
 test1.txt in the master branch. Add a file test.txt in the test branch. TEMP
 Update the file test1.txt in the test branch again.

---
 test.txt  |  1 +
 test1.txt | 14 ++++++++++++++
 2 files changed, 15 insertions(+)
 create mode 100644 test.txt
 create mode 100644 test1.txt

diff --git a/test.txt b/test.txt
new file mode 100644
index 0000000..8bd880f
--- /dev/null
+++ b/test.txt
@@ -0,0 +1 @@
+This is a file test.txt in the test branch.
diff --git a/test1.txt b/test1.txt
new file mode 100644
index 0000000..c465fcb
--- /dev/null
+++ b/test1.txt
@@ -0,0 +1,14 @@
+This is a file test1.txt.
+
+adsfasf
+Update something in the master branch.....
+
+asfasdfsadf.
+
+
+
+
+
+
+
+asdfsadfasdddddddddddddddddddddddd
-- 
1.9.5.msysgit.1

